### PR TITLE
1) Fix Swagger API Errors 2) Add examples references to swagger spec

### PR DESCRIPTION
**Pull request by sw47**
This checklist is used to make sure that common issues in a pull request are addressed. This will expedite the process of getting your pull request merged and avoid extra work on your part to fix issues discovered during the review process. 

### PR information
- [X ] The title of the PR is clear and informative.
- [X ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).
- [X ] Except for special cases involving multiple contributors, the PR is started from a fork of the main repository, not a branch.
- [x] If applicable, the PR references the bug/issue that it fixes.
- [ X] Swagger files are correctly named (e.g. the `api-version` in the path should match the `api-version` in the spec).

### Quality of Swagger
- [ X] I have read the [contribution guidelines](https://github.com/Azure/azure-rest-api-specs/blob/master/.github/CONTRIBUTING.md).
- [X ] My spec meets the review criteria:
  - [ ] The spec conforms to the [Swagger 2.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md).
  - [X ] The spec follows the guidelines described in the [Swagger checklist](../documentation/swagger-checklist.md) document.
  - [X ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

### CODE REVIEW

**Review by marstr — COMMENTED** [review]
Before we get started in earnest, I need understand the nature of the API Version `2015-11-01-preivew`. We currently categorize it as a `preview` version. Should this be considered more of a production or stable API Version? It's old enough, and still receiving enough attention that I suspect it is miscategorized.

**Comment by sw47** [comment]
@marstr Currently we don't have plans to remove the preview to move to a stable version as that will result in being upheld to the 99.5 SLA. I have brought this comment with my team. The scope the PR is to only fix the validation errors in Swagger spec for the Azure Fundamentals meeting.

File: specification/operationalinsights/resource-manager/Microsoft.OperationalInsights/preview/2015-11-01-preview/examples/LinkedServicesListByWorkspace.json
Comment (marstr) [inline specification/operationalinsights/resource-manager/Microsoft.OperationalInsights/preview/2015-11-01-preview/examples/LinkedServicesListByWorkspace.json]: If these are real subscription IDs, they should be obscured.

**Review by marstr — CHANGES_REQUESTED** [review]
In addition to my comments, because this PR adds operations to a `stable` API Version (2015-03-20) we'll also need ARM's sign-off.

File: specification/operationalinsights/resource-manager/Microsoft.OperationalInsights/preview/2015-11-01-preview/examples/LinkedServicesListByWorkspace.json
Comment (sw47) [reply specification/operationalinsights/resource-manager/Microsoft.OperationalInsights/preview/2015-11-01-preview/examples/LinkedServicesListByWorkspace.json]: Changed to 00000000-0000-0000-0000-00000000000

**Comment by ravbhatnagar** [comment]
@sw47 - New property getting added. This would require an updated api-version.

**Comment by sw47** [comment]
@ravbhatnagar - Thanks for your feedback!

Just want to clarify, are you referring to the "Operations" property added to stable 2015-03-20?
I believe all RPs need this property since it was one of the RPCViolations: Operations API must be implemented for '/providers/Microsoft.OperationalInsights/operations'. That is why I added the this property for this version without creating a new version.

Or are you referencing the "Features" property added to 2015-11-01-preview? I thought we don't need to bump this API version if it was still in preview. I can remove it if need be.
My goal here is to only suppress the SDK/RPC swagger errors. The future publishing of latest APIs and GAing a later version is still in discussion (active thread) since the work is being transitioned.

Can you please provide guidance on what I need only to fix the errors without introducing new versions? Thanks!

**Comment by ravbhatnagar** [comment]
@sw47 - All errors need to be fixed but going through the proper API-versioning. We should not go back to older api-versions and fix stuff in those. That will cause breaking changes in existing SDKs generated using that api-version. So in this case, it will be about adding the "features" model which is a part of workspace properties.

**Comment by sw47** [comment]
@ravbhatnagar - Thanks. As we discussed, I updated the swagger docs in the latest commit. Please take a look

**Comment by ravbhatnagar** [comment]
Signing off!

File: specification/operationalinsights/resource-manager/readme.md
Comment (marstr) [inline specification/operationalinsights/resource-manager/readme.md]: Just to confirm one last time before we merge: Was the original reference to this file a typo?

File: specification/operationalinsights/resource-manager/Microsoft.OperationalInsights/preview/2015-11-01-preview/OperationalInsights.json
Comment (marstr) [inline specification/operationalinsights/resource-manager/Microsoft.OperationalInsights/preview/2015-11-01-preview/OperationalInsights.json]: Was this case change due to the Linter alone? I was chatting with @lmazuel, and he mentioned that sometimes the linter scares people into doing the wrong thing in this circumstance. IF the service really does return `eTag` instead of `etag` we should add a suppression instead.

File: specification/operationalinsights/resource-manager/Microsoft.OperationalInsights/preview/2015-11-01-preview/OperationalInsights.json
Comment (marstr) [inline specification/operationalinsights/resource-manager/Microsoft.OperationalInsights/preview/2015-11-01-preview/OperationalInsights.json]: Seeing as this returns a list, it should be marked as `x-ms-pageable`. I can help hook you up with examples of how to do that if necessary.

**Review by marstr — CHANGES_REQUESTED** [review]
I just did one last pass, and found a few more quick things.

File: specification/operationalinsights/resource-manager/Microsoft.OperationalInsights/preview/2015-11-01-preview/OperationalInsights.json
Comment (sw47) [reply specification/operationalinsights/resource-manager/Microsoft.OperationalInsights/preview/2015-11-01-preview/OperationalInsights.json]: Added x-ms-pageable. Please see next commit

File: specification/operationalinsights/resource-manager/readme.md
Comment (sw47) [reply specification/operationalinsights/resource-manager/readme.md]: Yes, I believe so

File: specification/operationalinsights/resource-manager/Microsoft.OperationalInsights/preview/2015-11-01-preview/OperationalInsights.json
Comment (sw47) [reply specification/operationalinsights/resource-manager/Microsoft.OperationalInsights/preview/2015-11-01-preview/OperationalInsights.json]: You are right. I changed to be eTag and added suppression.

File: specification/operationalinsights/resource-manager/readme.md
Comment (marstr) [inline specification/operationalinsights/resource-manager/readme.md]: To leave some paper-trail: I was the one who called for this suppression.

File: specification/operationalinsights/resource-manager/Microsoft.OperationalInsights/preview/2015-11-01-preview/examples/OperationsList.json
Comment (marstr) [inline specification/operationalinsights/resource-manager/Microsoft.OperationalInsights/preview/2015-11-01-preview/examples/OperationsList.json]: Seems like some formatting got messed up in the most recent iteration. Mind moving back to two-space indentation?

**Review by marstr — CHANGES_REQUESTED** [review]
Thanks for responding so quickly! Last ask, it seems like some documents got mis-formatted this time around.

File: specification/operationalinsights/resource-manager/Microsoft.OperationalInsights/preview/2015-11-01-preview/examples/OperationsList.json
Comment (lmazuel) [reply specification/operationalinsights/resource-manager/Microsoft.OperationalInsights/preview/2015-11-01-preview/examples/OperationsList.json]: Look at the beginning of the file, there is now a "value" node, that indent everything further ;)

Comment (marstr) [reply specification/operationalinsights/resource-manager/Microsoft.OperationalInsights/preview/2015-11-01-preview/examples/OperationsList.json]: Whoops, nevermind then!

File: specification/operationalinsights/resource-manager/Microsoft.OperationalInsights/preview/2015-11-01-preview/examples/DataSourcesListByWorkspace.json
Comment (marstr) [inline specification/operationalinsights/resource-manager/Microsoft.OperationalInsights/preview/2015-11-01-preview/examples/DataSourcesListByWorkspace.json]: One more comment, I'm still seeing some mixed casing between `eTag` and `etag`. Let's resolve these all to be consistent with whichever way it is on the server.

Comment (sw47) [reply specification/operationalinsights/resource-manager/Microsoft.OperationalInsights/preview/2015-11-01-preview/examples/DataSourcesListByWorkspace.json]: I changed them all to eTag and updated the examples. Please see next iteration.

**Review by marstr — APPROVED** [review]

**Pull request merged**
